link/src/main/java/com/stripe/android/link/ui/wallet/WalletViewModel.kt

```diff
@@ -240,6 +240,33 @@ internal class WalletViewModel @Inject constructor(
         navigator.navigateTo(LinkScreen.CardEdit(paymentDetails.id))
     }
 
+    fun setDefault(paymentDetails: ConsumerPaymentDetails.PaymentDetails) {
+        _uiState.update {
+            it.copy(paymentMethodIdBeingUpdated = paymentDetails.id)
+        }
+
+        viewModelScope.launch {
+            val updateParams = ConsumerPaymentDetailsUpdateParams.Card(
```

Comment:
Yeah, we just wouldn’t set `cardPaymentMethodCreateParams`. I will remove the `Card` subclass though to make things more obvious.

Also, on iOS, we use `PaymentMethodParams` instead of `PaymentMethodCreateParams`. Should we move Android to that as well?

Reply:
`PaymentMethodCreateParams` is the type, I think leaving it is clearer. That's an optional parameter only for cards, so maybe the `Card` subclass can have a constructor with that parameter, and the superclass takes only `id` and `isDefault`.

Reply:
I’m not sure that I fully understand. My approach for now is to have a single `ConsumerPaymentDetailsUpdateParams` data class that takes in `id`, `isDefault`, and an optional `cardPaymentMethodCreateParams`. Is that in line with what you mean?